### DISCUSSION TITLE
Is there a android app/way  that can sync with foliate?

### DISCUSSION THREAD

**Discussion by HGqK8aekLaPfGB** [johnfactotum/foliate#737] [Q&A] [answered]
I prefer this over googles play books, but now I can't read on android/Linux  simultaneously

**Comment by johnfactotum** [comment]
You can try the method detailed in #690 by @apandada1.

**Comment by archisman-panigrahi** [comment, marked as answer]
If you use the method in #690, you can use git to sync the config directory between the PC and the phone. Unless the traffic/directory-size is too high, GitHub probably won't be unhappy.
